### PR TITLE
Proper parenthetics

### PR DESCRIPTION
**Pull request by iamrobinhood12345**
Can improve by refactoring Doubly_Linked_List to remove from tail to head instead so that the text parameter does not have to be reversed.

### CODE REVIEW

**Review by iamrobinhood12345 — COMMENTED** [review]
Nice

File: proper_parenthetics.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from doubly_linked_list import Doubly_Linked_List
+
+def proper_parenthetics(text):
+	"""Returns a 1 if the string is parenthetically open.
+	Returns a 0 if the string is parenthetically balanced.
+	Returns a -1 if the string is parenthetically broken."""
+	parenthetics = Doubly_Linked_List()
+	text = text[::-1]
+	for char in text:
+		if char == '(' or char == ')':
```
Comment (nhuntwalker) [inline proper_parenthetics.py:14]: `if char in ["(", ")"]`

Indentation here is waaaaaay too large. Are you using actual `tab` characters? Or are you using 4 spaces like you were supposed to.

File: test_proper_parenthetics.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import pytest
+
+PARAMS_PAREN = [
+	('()()()()', 0),
+	(')()()()', -1),
+	('()()()(', 1),
+	('()((()', 1),
+	('()))()', -1),
+	('))((', -1),
```
Comment (nhuntwalker) [inline test_proper_parenthetics.py:11]: You've gotta try more examples of nested parens. Don't worry about text that isn't parenthesis characters

File: proper_parenthetics.py (hunk 1 above)
Comment (iamrobinhood12345) [reply proper_parenthetics.py:14]: its was a tab. i think i hadn't set up tabs to spaces in sublime text.

File: test_proper_parenthetics.py (hunk 2 above)
Comment (iamrobinhood12345) [reply test_proper_parenthetics.py:11]: Okay cool

**Pull request merged**